### ISSUE TITLE
Add releases to repo

### ISSUE THREAD

**Issue by vladimirvivien** [vmware-tanzu/sonobuoy-plugins#35]
Add formal releases to repository.

**Comment by johnSchnake** [comment]
With the idea of plugin management, ideally it seem like either we would have a new release for _all_ plugins each time any given one changes, so it would just be the SHA effectively.

I think we need to flush out the ideas around the plugin management more before we know the appropriate release strategy for the repo. IMO it sounds like we may not need releases for this as a repo as a whole. May also get some ideas from how things like brew handle releases/versions for all the different tools the support.

**Comment by vladimirvivien** [comment]
@johnSchnake agreed. Again, this was early days of me thinking about this. I think we can rethink the approach to do something better. The plugin management will dictate how plugins are hosted and released (mono vs decoupled repos).
